src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs

```diff
@@ -231,17 +237,22 @@ private static async Task<Bitmap> DecodeStreamAsBitmapWithExifOrientation(Stream
 			stream.Position = 0;
 
 			var exifInterface = new ExifInterface(stream);
+
 			return RespectExifOrientation(exifInterface, bitmap);
 		}
 
-		private static async Task<Bitmap> DecodeFileAsBitmapWithExifOrientation(string path, BitmapFactory.Options options = null)
+		private static async Task<Bitmap?> DecodeFileAsBitmapWithExifOrientation(string path, BitmapFactory.Options? options = null)
 		{
-			var bitmap = await BitmapFactory.DecodeFileAsync(path, options);
+			if (await BitmapFactory.DecodeFileAsync(path, options) is not { } bitmap)
+			{
+				return null;
+			}
+
 			var exifInterface = new ExifInterface(path);
 			return RespectExifOrientation(exifInterface, bitmap);
 		}
 
-		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView targetImage = null, int? targetWidth = null, int? targetHeight = null)
+		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView targetImage, int? targetWidth = null, int? targetHeight = null)
```

Comment:
The caller in ImageBrush passes null for `targetImage`, so I think it should be nullable. `ImageBrush` doesn't have `#nullable enable` so the build might be passing.

```suggestion
		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView? targetImage, int? targetWidth = null, int? targetHeight = null)
```

Reply:
Thanks. Updated. This change also required updating a delegate signature.